### DISCUSSION TITLE
TLE parameters for Artemis

### DISCUSSION THREAD

**Discussion by AdoMassa** [mworion/MountWizzard4#266] [Q&A] [answered]
In the next few years, there will be a few massive rockets to be launch in orbit that can be captured with amateur telescopes. In particular the Artemis mission (next Monday) but also Starship from Space X.

It is impossible to have accurate TLE parameter priors to the launch and the value of the TLE must be updated constantly as it changes due to launch delays as well as variation of the trajectory of the rockets.

What would be the organization to ask updated TLE parameter for these rockets? 

I don't know how many orbits Starship and Artemis will do before to return to Earth or to be launch to the moon. Either way, it would be amazing to build an international network of telescope to follow the mission.

**Comment by mworion** [comment, marked as answer]
Hi,
I would guess that organizations like CELESTRAK will publish these updates in their database. There might be others as well.
Michel
